### ISSUE TITLE
[Feature]: Create card from template

### ISSUE THREAD

**Issue by n0bra1n3r** [mgmeyers/obsidian-kanban#493]
### Goal or desired outcome of this feature

More automation.

### Describe the feature

Ability to create a card using a template note defined in a folder, in addition to the "Add a card" button. Could integrate with the Templater plugin for example.

### Can you think of any alternatives or work-arounds?

The "Duplicate card" operation could work as a work-around.

**Comment by Archie-2021** [comment]
+1 for this one

**Comment by lovwyr** [comment]
Defining an Template per List where also be an cool Feature :)

**Comment by ThomJ130** [comment]
Yeah, I agree. I have a standard format for all of my cards including header level and the date for the title, but templates are only supported for note creation, not card creation. A bare minimum for this is simply a `Default Card Template`, where when you create a card in any list it uses the default template. There is one issue: where does the cursor start? It would be nice to have control over this, which in VS Code is simply $0 to indicate the starting position of the cursor.

Please add this!
